sim/pkg/serve/index.html

```diff
@@ -479,9 +510,19 @@ <h1 class="title">Skenario</h1>
         let requestTimeoutSec = parseInt(document.querySelector("input[id='requestTimeoutSec']").value);
         let requestCPUTimeMillis = parseInt(document.querySelector("input[id='requestCPUTimeMillis']").value);
         let requestIOTimeMillis = parseInt(document.querySelector("input[id='requestIOTimeMillis']").value);
+        let plugins = {};
+
         let hpaEnabled = document.querySelector("input[id='hpa']").checked;
         let hpaYaml = document.querySelector("textarea[id='hpaYaml']").value;
+        let vpaEnabled = document.querySelector("input[id='vpa']").checked;
+        let vpaYaml = document.querySelector("textarea[id='vpaYaml']").value;
 
+        if (hpaEnabled) {
+            plugins['hpa.v2beta2.autoscaling.k8s.io'] = hpaYaml;
+        }
+        if (vpaEnabled) {
+            plugins['vpa.v2beta2.autoscaling.k8s.io'] = vpaYaml;
```

Comment:
Should be `v1` not `v2beta2`: 

https://github.com/kubernetes/autoscaler/blob/d9e8fa3cbba3c3c76adf96e70bbb1277fad60ea7/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/register.go#L26

Reply:
Fixed it.